src/utils.rs

```diff
@@ -40,6 +40,16 @@ pub fn currency_from_network(network: Network) -> Currency {
     }
 }
 
+pub fn network_from_currency(currency: Currency) -> Network {
+    match currency {
+        Currency::Bitcoin => Network::Bitcoin,
+        Currency::BitcoinTestnet => Network::Testnet,
+        Currency::Signet => Network::Signet,
+        Currency::Regtest => Network::Regtest,
+        Currency::Simnet => Network::Regtest,
```

Comment:
is `currency_from_network` not going to break something if there's no `Regtest` to map to `Simnet` or is that the reason for `is_valid_network` everywhere?

Reply:
Simnet is just btcd's version of regtest, we don't really need to worry about it.